### ISSUE TITLE
pcap initialization in pcap/pcap.go

### ISSUE THREAD

**Issue by GoogleCodeExporter** [scroot/gopacket#7]
```
pcap can't replay packets with WritePacketData because of the way pcap is 
initialized in OpenLive. I've switched that pcap_create and subsequent code 
with pcap_open_live() and WritePacketData worked as expected.

```

Original issue reported on code.google.com by `fio...@gmail.com` on 30 Mar 2014 at 8:54

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 
https://code.google.com/p/gopacket/source/detail?r=b3c07bdf914dd512a57899ef1c9ff
2c37bf2c53f

Fixed library version tag is v1.0.10
```

Original comment by `gconnell@google.com` on 31 Mar 2014 at 4:30
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Thank you.
```

Original comment by `fio...@gmail.com` on 31 Mar 2014 at 4:33
